onnx/backend/test/runner/__init__.py

```diff
@@ -165,7 +165,7 @@ def test_suite(self) -> unittest.TestSuite:
             unittest.TextTestRunner().run(BackendTest(backend).test_suite)
         """
         suite = unittest.TestSuite()
-        for case in sorted(self.test_cases.values()):
+        for case in sorted(self.test_cases.values()):  # type: ignore
```

Comment:
May I understand which kind of error does it ignore?

Reply:
I assume TestCase does not really support sort. I'll check.

Reply:
Changed.